gradle/libs.versions.toml

```diff
@@ -1,6 +1,9 @@
 # keep sorted please, use Edit -> Sort Lines
 
 [versions]
+androidCompileSdk = "32"
+androidMinSdk = "21"
+androidTargetSdk = "32"
```

Comment:
You can use the same version for `compile` and `target`, I do not think we will have time when these 2 will differ.

Reply:
Hopefully we won't, but at least it's centralised here, and if we do need to for some reason, we have the flexibility to do so.